pkg/commands/patch/patch_test.go

```diff
@@ -445,7 +443,7 @@ func TestModifyPatchForRange(t *testing.T) {
 			reverse:        true,
 			expected: `--- a/newfile
 +++ b/newfile
-@@ -0,0 +1,1 @@
+@@ -0,0 +1 @@
```

Comment:
Unified diff allows us to omit the new length if it's value is 1. I'm doing so here purely because it makes testing more straightforward.